src/tickit/core/management/event_router.py

```diff
@@ -121,7 +121,9 @@ def from_component_configs(
         Returns:
             InverseWiring: A mapping of component input ports to component output ports.
         """
-        return cls({config.name: config.inputs for config in configs})
+        wiring = cls({config.name: config.inputs for config in configs})
```

Comment:
Return directly, prevents InverseWiring named wiring

Reply:
I already did this but while you were reviewing, oops.